libr/util/table.c

```diff
@@ -278,17 +278,20 @@ static void __computeTotal(RTable *t) {
 }
 
 R_API char *r_table_tofancystring(RTable *t) {
+	if (r_list_length (t->cols) == 0) {
+		return NULL;
```

Comment:
What about `""`? I would expect NULL to be returned when there are errors, while no columns looks more like a valid table, just empty.

Reply:
more about strdup(""), but as long as strdup can also return NULL, i think it's fine to return null in case there's nothing to be printed or there's an error

Reply:
I think there is a semantic difference between return NULL or "". NULL means there was an error (if strdup returns NULL it means it can't allocate memory, so it is definitely an error), while "" means all was good, but there is nothing to print. As a user of the `r_table_tofancystring` API it is important to distinguish the two things.

Reply:
done

Reply:
done where?

Reply:
working with 3 ghusers confuses me sometimes, i pushed this into the trufae fork. its now in here